nixos/modules/services/web-apps/mealie.nix

```diff
@@ -0,0 +1,145 @@
+
+      DB_ENGINE = lib.mkDefault "sqlite";
+      SECURITY_MAX_LOGIN_ATTEMPTS = lib.mkDefault "5";
+      SECURITY_USER_LOCKOUT_TIME = lib.mkDefault "24";
+
+      POSTGRES_USER = lib.mkDefault "mealie";
+      POSTGRES_PASSWORD = lib.mkDefault "mealie";
+      POSTGRES_SERVER = lib.mkDefault "postgres";
+      POSTGRES_PORT = lib.mkDefault 5432;
+      POSTGRES_DB = lib.mkDefault "mealie";
+
+      SMTP_PORT = lib.mkDefault 587;
+      SMTP_FROM_NAME = lib.mkDefault "mealie";
+      SMTP_AUTH_STRATEGY = lib.mkDefault "TLS";
+
+      WEB_GUNICORN = lib.mkDefault "false";
+      WORKERS_PER_CORE = lib.mkDefault 1;
+      MAX_WORKERS = lib.mkDefault 1;
+      WEB_CONCURRENCY = lib.mkDefault 1;
+
+      LDAP_AUTH_ENABLED = lib.mkDefault "False";
+      LDAP_TLS_INSECURE = lib.mkDefault "False";
+      LDAP_ENABLE_STARTTLS = lib.mkDefault "False";
+      LDAP_ID_ATTRIBUTE = lib.mkDefault "uid";
+      LDAP_NAME_ATTRIBUTE = lib.mkDefault "name";
+      LDAP_MAIL_ATTRIBUTE = lib.mkDefault "mail";
+
+      THEME_LIGHT_PRIMARY = lib.mkDefault "#E58325";
+      THEME_LIGHT_ACCENT = lib.mkDefault "#007A99";
+      THEME_LIGHT_SECONDARY = lib.mkDefault "#973542";
+      THEME_LIGHT_SUCCESS = lib.mkDefault "#43A047";
+      THEME_LIGHT_INFO = lib.mkDefault "#1976D2";
+      THEME_LIGHT_WARNING = lib.mkDefault "#FF6D00";
+      THEME_LIGHT_ERROR = lib.mkDefault "#EF5350";
+      THEME_DARK_PRIMARY = lib.mkDefault "#E58325";
+      THEME_DARK_ACCENT = lib.mkDefault "#007A99";
+      THEME_DARK_SECONDARY = lib.mkDefault "#973542";
+      THEME_DARK_SUCCESS = lib.mkDefault "#43A047";
+      THEME_DARK_INFO = lib.mkDefault "#1976D2";
+      THEME_DARK_WARNING = lib.mkDefault "#FF6D00";
+      THEME_DARK_ERROR = lib.mkDefault "#EF5350";
+    };
+
+    systemd.services.mealie = {
+      description = "Mealie, a self hosted recipe manager and meal planner";
+      after = [
+        "network.target"
+        "network-online.target"
+      ];
+      wantedBy = [
+        "multi-user.target"
+      ];
+
+      environment = {
+        PYTHONPATH = "${pkg.pythonPkg.pkgs.makePythonPath pkg.propagatedBuildInputs}:${pkg}/lib/mealie"; #${pkg.python.libPrefix}/site-packages";
+        PRODUCTION = "true";
+
+        # Data directory defined by systemd service
+        MEALIE_LOG_FILE = "/var/log/mealie/mealie.log";
+        ALEMBIC_CONFIG_FPATH="/var/lib/mealie/alembic.ini";
+
+        # Configuration from module
+        LOG_LEVEL = cfg.logLevel;
+        API_PORT = builtins.toString cfg.port;
+        BASE_URL = "${cfg.protocol}://${cfg.host}:${builtins.toString cfg.port}";
+      } // (builtins.mapAttrs (_: val: builtins.toString val) cfg.extraConfig);
+
+      serviceConfig = {
+        DynamicUser = true;
+        User = "mealie";
+        ExecStartPre = pkgs.writeShellScript "startup-mealie.sh" ''
+          ${lib.getExe pkgs.gnused} 's+script_location = alembic+script_location = ${pkg.src}/alembic+g' ${pkg.src}/alembic.ini > $ALEMBIC_CONFIG_FPATH
```

Comment:
Rather than be a `sed` invocation in the service, this patch should be done in the package itself (e.g: in the `postPatch` phase) and the ini packaged up somewhere in `$out` (I don't have a good idea for the exact location yet, I'll think about it).